src/member/controller/MemberController.ts

```diff
@@ -69,15 +69,41 @@ export class MemberController {
 
     return { data, links, code: 201 };
   }
+
+  /**
+   * Update a Member
+   */
+  public async put(request: MemberPutRequest): Promise<MemberResponse> {
+    const links = {};
+    const member = await this.getModel(request.id);
+    if (member) {
+      member.default_transport_mode = request.defaultTransportMode;
+      member.default_distance = request.defaultDistance;
+    }
+
+    const savedModel = await this.genericRepository.save(Object.assign(member, MemberView));
```

Comment:
what's the story with this Object.assign?

Reply:
getModel returns Amember which is different the Member model even tho its the same